pkg/scheduler/framework/runtime/framework.go

```diff
@@ -788,7 +788,7 @@ func (f *frameworkImpl) runPostBindPlugin(ctx context.Context, pl framework.Post
 // RunReservePluginsReserve runs the Reserve method in the set of configured
 // reserve plugins. If any of these plugins returns an error, it does not
 // continue running the remaining ones and returns the error. In such a case,
-// the pod will not be scheduled.
+// the pod will not be scheduled and Unreserve will be called for all plugins.
```

Comment:
uhmm... this function doesn't actually call Unreserve. The comment could say:
"In such a case, the pod willl not be scheduled and RunReservePluginsUnreserve should be called afterwards"

Reply:
ugh, I forgot that I changed my implementation; remember when it used to be such that RunReservePluginsReserve would call RunReservePluginsUnreserve :D

good catch :)